src/app/components/home.component.ts

```diff
@@ -1098,6 +1106,31 @@ export class HomeComponent implements OnInit, AfterViewInit {
     this.scrollToTop();
   }
 
+  /**
+   * Scroll appropriately after navigating back to root folder
+   *
+   * Rather hacky thing, but works in the basic case
+   * Fails if user enters folder, changes some search or sort filter, and navigates back
+   */
+  scrollAppropriately(filter: string) {
+    if (filter === '') {
+      // TODO -- change `1` to `0` once the bug is fixed:
+      // https://github.com/rintoj/ngx-virtual-scroller/issues/420
+      this.virtualScroller.scrollToPosition(this.folderNavigationScrollOffset, 1);
```

Comment:
Will need to change this to `0` once the bug is fixed:
https://github.com/rintoj/ngx-virtual-scroller/issues/420